### DISCUSSION TITLE
"Odwrotne" dzialanie systemu generowania i pobierania etykiet.

### DISCUSSION THREAD

**Discussion by 889konrad889** [allegro/allegro-api#5009] [Suggestion - verification]
Jest ewidetny błąd przy wyborze rodzaju etykiety w fazie projektowania funkcji odpowiedzialnych za generowanie paczek i pobieranie etykiet, możliwe że osoba odpowiedzialna za to nie miała doczynienia z niezbędnością takiej funkcjonalności wcześniej i tego nie przewidziała.

Wszystkie synchronizacje APi z którymi miałem doczynienia: Inpost, Gls, DPD, posiada funkcjonalność wyboru rodzaju etykiety przy jej pobieraniu niezależnie od jej rodzaju podanego przy generowaniu.

"text/plain" zajmuje mniej zasobów, przy generowaniu oraz transferze niż PDF.

Brak możliości wybodu rodzaju pobieranej etykiety powoduje spore kłopoty:
Generowanie etykiet np z poziomu Eazy Uploadera na jednym stanowisku,
następnie gdy z bazy oline'owy system pakowania oparty na PHP pobierze etykiete przez API chce wysłać ją na drukarke zebry.
a na innym stanowisku pakowania z drukarką datamaxa.

Uprzejmie proszę o zmianę struktury zapytanie dodając odpowiedni parametr.

**Comment by hahyes** [comment]
Drobna poprawka, którą zgłosiłem swoją drogą. Nie ma czegoś takiego jak typ `plain/text`. Jest tylko `text/plain`. :)

**Reply by 889konrad889** [reply]
Dzieki pisałem przed poranną kawą na telefonie,  już wpis poprawiłem :D
